client-tez/src/main/java/org/apache/tez/runtime/library/common/shuffle/impl/RssTezFetcherTask.java

```diff
@@ -195,15 +190,15 @@ protected FetchResult callInternal() throws Exception {
                       .hadoopConf(hadoopConf)
                       .idHelper(new TezIdHelper())
                       .expectedTaskIdsBitmapFilterEnable(expectedTaskIdsBitmapFilterEnable)
-                      .rssConf(RssTezConfig.toRssConf(this.conf)));
+                      .rssConf(this.conf.toRssConf()));
```

Comment:
Same with toRssConf. Can we pass this.conf directly?

Reply:
I think not, the RSS_STORAGE_TYPE in TezClientConf is 'tez.rss.storage.type',

```
  public static final ConfigOption<String> RSS_STORAGE_TYPE =
      ConfigOptions.key("tez.rss.storage.type")
          .stringType()
          .defaultValue("MEMORY_LOCALFILE")
          .withDescription("");
```

But in ShuffleReadClientImpl.java, when get storageType, it call:
`final String storageType = builder.getRssConf().get(RssClientConf.RSS_STORAGE_TYPE);`